src/test/kotlin/se/uulm/snowballr/backend/service/project/RemoveProjectMemberTest.kt

```diff
@@ -0,0 +1,165 @@
+
+        assertDoesNotThrow { mainService.removeProjectMember(getExampleRequest()) }
+    }
+
+    @Test
+    fun `When a project admin removes themselves and they are not the last project admin, then no exception is thrown`() =
+        runTest {
+            val currentUser = DataBuilder.createExampleUser(id = userId)
+            val otherUser = DataBuilder.createExampleUser()
+            val project = DataBuilder.createExampleProject(id = projectId)
+            val projectAdmin1 = DataBuilder.createExampleProjectMember(projectId = project.id, userId = userId)
+            val projectAdmin2 = DataBuilder.createExampleProjectMember(projectId = project.id, userId = otherUser.id)
+
+            mockCurrentUser(currentUser)
+            coEvery {
+                projectMemberRepoMock.getProjectMembers(project.id)
+            } returns listOf(projectAdmin1, projectAdmin2)
+            coEvery {
+                projectMemberRepoMock.getAllProjectAdmins(project.id)
+            } returns listOf(projectAdmin1, projectAdmin2)
+            coEvery { projectRepoMock.doesProjectExistById(project.id) } returns true
+            coEvery { userRepoMock.doesUserExistById(currentUser.id) } returns true
+            coEvery {
+                projectMemberRepoMock.removeProjectMember(project.id, currentUser.id)
+            } returns Unit
+
+            assertDoesNotThrow { mainService.removeProjectMember(getExampleRequest()) }
+        }
+
+    @Test
+    fun `When a non project admin removes themselves and they are not the last project member, then no exception is thrown`() =
+        runTest {
+            val currentUser = DataBuilder.createExampleUser(id = userId)
+            val otherUser = DataBuilder.createExampleUser()
+            val project = DataBuilder.createExampleProject(id = projectId)
+            val projectMember = DataBuilder.createExampleProjectMember(projectId = project.id, userId = userId)
+            val projectAdmin = DataBuilder.createExampleProjectMember(projectId = project.id, userId = otherUser.id)
+
+            mockCurrentUser(currentUser)
+            coEvery {
+                projectMemberRepoMock.getProjectMembers(project.id)
+            } returns listOf(projectMember, projectAdmin)
+            coEvery { projectMemberRepoMock.getAllProjectAdmins(project.id) } returns listOf(projectAdmin)
+            coEvery { projectRepoMock.doesProjectExistById(project.id) } returns true
+            coEvery { userRepoMock.doesUserExistById(currentUser.id) } returns true
+            coEvery {
+                projectMemberRepoMock.removeProjectMember(project.id, currentUser.id)
+            } returns Unit
+
+            assertDoesNotThrow { mainService.removeProjectMember(getExampleRequest()) }
+        }
+
+    @Test
+    fun `When a project admin removes a project member from a non-existing project, then a NotFoundException is thrown`() =
```

Comment:
```suggestion
    fun `When a project admin removes a project member from a nonexistent project, then a NotFoundException is thrown`() =
```
I think we corrected it everywhere to this style, if possible I would stick to it to be consistent.

Reply:
Right. Thanks.